app/models/operatingsystems/redhat.rb

```diff
@@ -72,4 +76,21 @@ def pxe_kernel_options(params)
   def minor_version_help
     '0, 6.1810'
   end
+
+  def fedora?
+    name == 'Fedora'
+  end
+
+  def rhel_compatible?
+    name != 'Fedora'
+  end
+
+  # Tells if the OS meets a requirement. The requirement can be specified as rhel version, fedora version or both.
+  def meets_requirement(fedora: nil, rhel: nil)
+    # prefer no translation
+    return major.to_i >= fedora if fedora && fedora?
+    return major.to_i >= rhel if rhel && rhel_compatible?
+
+    false
```

Comment:
I'm not a fan of guard clauses. Is this better?
```suggestion
    (fedora && fedora? && major.to_i >= fedora) || (rhel && rhel_compatible? && major.to_i >= rhel)
```

Reply:
IMHO my version is a bit more readable, but we will need tiebreaker here. @stejskalleos your opinion?

Reply:
Personally, I found the multi-line solution more readable.